exp/api/v1beta1/azuremanagedcontrolplane_webhook.go

```diff
@@ -270,7 +270,7 @@ func (m *AzureManagedControlPlane) Validate() error {
 }
 
 // validate DNSServiceIP.
```

Comment:
Nit: capitalize "validate" in comment to be consistent

Reply:
I fixed the comment to be consistent with the function name, and fixed the others that were using inconsistent capitalization